docs/providers/aws/guide/iam.md

```diff
@@ -330,4 +331,122 @@ resources:
                     - ec2:DetachNetworkInterface
                     - ec2:DeleteNetworkInterface
                   Resource: "*"
+                 
+```
+
```

Comment:
This is the wrong section for this bit of documentation. This section is about the IAM roles for your functions which is different from the role and policy that the framework/cloud formation uses to perform the deployment. We should actually put this into the credentials section 
https://github.com/serverless/serverless/blob/6c7eada33b698e93f2725457ecccb12a93fd53d5/docs/providers/aws/guide/credentials.md#creating-aws-access-keys